src/main/scala/components/Core.scala

```diff
@@ -3,9 +3,9 @@ package nucleusrv.components
 import chisel3._
 import chisel3.util._
 
-class Core(M:Boolean = false) extends Module {
+class Core(M:Boolean = false, RVFI:Boolean=false, XLEN:Int) extends Module {
```

Comment:
write default value XLEN it should compile in case user didn't provide it in instance.

Reply:
ok